android/cli/commands/_buildModule.js

```diff
@@ -712,9 +743,7 @@ AndroidModuleBuilder.prototype.processResources = function processResources(next
 		function generateAaptFriendlyManifest(cb) {
 			var manifestTemplatePathAndFilename = path.join(this.moduleGenTemplateDir, 'AndroidManifest.xml.ejs');
 			var manifestOutputPathAndFilename = path.join(this.buildIntermediatesDir, 'manifests/aapt/AndroidManifest.xml');
-			if (!fs.existsSync(path.dirname(manifestOutputPathAndFilename))) {
-				wrench.mkdirSyncRecursive(path.dirname(manifestOutputPathAndFilename));
-			}
+			fs.ensureDirSync(manifestOutputPathAndFilename);
```

Comment:
Not sure what's the expected behavior if you omit `path.dirname`? Probably better to take it over.

Reply:
I thought it should be able to because the other fs-references do it the same way, but let's keep it as it was before for now.